### PR TITLE
Publish image to AuroraProdACR

### PR DESCRIPTION
**Pull request by tsundvoll**

### CODE REVIEW

File: .github/workflows/publish_docker_image.yml [hunk 1]
```diff
@@ -5,11 +5,13 @@ on:
     types: [published]
 
 env:
-  REGISTRY: ghcr.io
-  IMAGE_NAME: ${{ github.repository }}
+  GitHubREGISTRY: ghcr.io
+  GitHubIMAGE_NAME: ${{ github.repository }}
+  AURORA_PROD_REGISTRY: auroraprodacr.azurecr.io
+  AURORA_PROD_IMAGE_NAME: /robotics/flotilla
```
Comment (GodVenn) [inline .github/workflows/publish_docker_image.yml:11]:
```suggestion
  GITHUB_REGISTRY: ghcr.io
  GITHUB_IMAGE_NAME: ${{ github.repository }}
  AURORA_PROD_REGISTRY: auroraprodacr.azurecr.io
  AURORA_PROD_IMAGE_NAME: /robotics/flotilla
```

**Review by GodVenn — COMMENTED** [review]
This looks good to me, although I don't understand the details of the Aurora pushes. I just have a comment on the naming convention

File: .github/workflows/publish_docker_image.yml (hunk 1 above)
Comment (tsundvoll) [reply .github/workflows/publish_docker_image.yml]: Good comment, resolved

Comment (tsundvoll) [reply .github/workflows/publish_docker_image.yml]: Nothing fancy about the Aurora pushes, just an internal place to store the containers

**Review by GodVenn — APPROVED** [review]
LGTM

**Pull request merged**